porch/repository/pkg/git/commit.go

```diff
@@ -274,14 +287,19 @@ func storeBlob(store storer.EncodedObjectStorer, value string) (plumbing.Hash, e
 		return plumbing.Hash{}, err
 	}
 
-	_, err = w.Write(data)
-	w.Close()
-	if err != nil {
+	if _, err := w.Write(data); err != nil {
 		return plumbing.Hash{}, err
```

Comment:
I _think_ this return will skip close on line 294

Reply:
Good call - thank you!